lib/inaturalist/observation.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Inaturalist
+  class Observation
+    def initialize(obs)
+      @obs = obs
+    end
+
+    def external_id
+      @obs['id']
+    end
+
+    def metadata
+      @metadata ||= extract_metadata(@obs)
+    end
+
+    def extract_metadata(obs)
+      metadata = {}
+      metadata['id'] = obs['id']
+      metadata['change'] = 'No changes were made to this image.'
```

Comment:
Curious what is this `metadata['change'] set` is for?

Reply:
This is boilerplate legal language that indicates that the image was not altered since it was uploaded by the iNat user. It has been standard on all manual iNat subject uploads.